### DISCUSSION TITLE
希望大家可以在这个目录下分享有用的云函数

### DISCUSSION THREAD

**Discussion by maslow** [labring/laf#201] [Share & Show]
Github laf 论坛（Disscussions)，

在 “Functions - 云函数分享” 主题目录下，新建帖子分享云函数。

发扬我为人人，人人为我的开源精神👍

建议一个帖子一个函数，或者多个有关联的函数，方便检索。

云函数最好上下文简单，方便大家修改使用。

（发帖时也可以考虑留个收款码，方便接受大家的谢意🧧

**Comment by virgoone** [comment]
微信小程序登陆注册

![image](https://user-images.githubusercontent.com/8540582/180955058-7834a68a-82a1-4d26-8280-279f0479d80d.png)

**Reply by maslow** [reply]
👍 建议新开贴分享哈
